plugins/mobile-plugin/src/test/java/pagefactory/mobile/supports/Utils.java

```diff
@@ -0,0 +1,18 @@
+package pagefactory.mobile.supports;
+
+import java.util.List;
+import java.util.Random;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import ru.sbtqa.tag.pagefactory.environment.Environment;
+
+public class Utils {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)